provisioning/miniprov/hare_mp/types.py

```diff
@@ -108,7 +108,7 @@ class NodeDesc(DhallTuple):
     data_iface: Text
     data_iface_type: Maybe[Protocol]
     m0_servers: Maybe[DList[M0ServerDesc]]
-    s3_instances: int
+    instances: int
```

Comment:
Not sure that we should rename s3_instances -> instances.

In the context of ConfStore the word `instances` is understood (just because field `instances` goes in scope of `s3` tree). But when I look at `NodeDesc`, I have no clue what `instances` stands for.

Reply:
Added new changes into that.